hl7v23/Dependencies.toml

```diff
@@ -123,7 +123,7 @@ modules = [
 [[package]]
 org = "ballerinax"
 name = "health.hl7v23"
-version = "1.0.6"
+version = "2.0.0"
```

Comment:
why is this 200 not 201?

Reply:
In the automated post-release commit, the bot bumps the version to the next dev version only in the bal toml. After that, the pkg is not built. Hence, this is having the version from the previous release commit.

Reply:
built the pkg and updated the version.